packages/taquito-beacon-wallet/package.json

```diff
@@ -77,6 +77,7 @@
     "@types/jest": "^26.0.23",
     "@types/libsodium-wrappers": "0.7.9",
     "@types/node": "^17.0.0",
+    "@types/sinon": "^10.0.13",
```

Comment:
@types/sinon doesn't seem to be used. Can it be removed from the dependencies?

Reply:
It has been removed.